src/pages/productListing/ProductListing.jsx

```diff
@@ -0,0 +1,166 @@
+              type="button"
+              className="icon-button"
+            >
+              <i className="fas fa-heart fa-3x"></i>
+              <span className="icon-button__badge">2</span>
+            </button>
+          </div>
+        </div>
+      </nav>
+
+      <div className="content product-grid">
+        <div className="filters">
+          <div className="filter-heading">
+            <h3 className="filter-head main">Filters</h3>
+            <p className="filter-head clear">Clear</p>
+          </div>
+          <form className="filter-form">
+            <div className="filter-category --price">
+              <label htmlFor="price">
+                <h3 className="filter-head">Price</h3>
+              </label>
+              <input
+                type="range"
+                name="price"
+                id="price"
+                min="0"
+                max="500000"
+                value="50"
+              />
+            </div>
+            <div className="filter-category --category">
+              <h3 className="filter-head">Category</h3>
+              <input type="checkbox" id="men" name="men" value="men" />
+              <label htmlFor="men">Men Clothing</label>
+              <br />
+              <input type="checkbox" id="women" name="women" value="women" />
+              <label htmlFor="women">Women Clothing</label>
+            </div>
+            <div className="filter-category --Rating">
+              <h3 className="filter-head">Rating</h3>
+              <input
+                type="radio"
+                id="four"
+                name="rating"
+                value="4 Stars and above"
+              />
+              <label htmlFor="four">4 Stars and above</label>
+              <br />
+              <input
+                type="radio"
+                id="three"
+                name="rating"
+                value="3 Stars and above"
+              />
+              <label htmlFor="three">3 Stars and above</label> <br />
+              <input
+                type="radio"
+                id="two"
+                name="rating"
+                value="2 Stars and above"
+              />
+              <label htmlFor="two">2 Stars and above</label>
+              <br />
+              <input
+                type="radio"
+                id="one"
+                name="rating"
+                value="1 Stars and above"
+              />
+              <label htmlFor="one">1 Stars and above</label>
+            </div>
+            <div className="filter-category --Sortby">
+              <h3 className="filter-head">Sort By</h3>
+              <input
+                type="radio"
+                id="lowtohigh"
+                name="sortby"
+                value="Low to High"
+              />
+              <label htmlFor="lowtohigh">Low To High</label>
+              <br />
+              <input
+                type="radio"
+                id="hightolow"
+                name="sortby"
```

Comment:
Camelcasing is missing here. Its suggested to use everywhere in code for good practises